tests/ConferenceTest.php

```diff
@@ -137,4 +146,63 @@ function guests_cannot_create_conference()
         $this->visit('conferences/create')
             ->seePageIs('login');
     }
+
+    /** @test */
+    function notification_gets_send_to_all_users_for_open_cfp_today()
```

Comment:
I think I may want to create a separate test about notifications.
Then I'd likely name this test something like:

`if_approved_conference_is_created_users_are_notified` or something. That said, the test itself will probably be a bit different anyway once we have the whole approval concept, since no conference will be approved when it's created except by an importer.